Safor/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="style.css" rel="stylesheet">
+        <title>Document</title>
+    </head>
+    <body>
+        <section class="topLayer">
```

Comment:
Strange naming of the section, please use dashes or underscores to separate different words.